src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -93,11 +101,25 @@ internal abstract partial class DefinitionItem
                 nameDisplayParts, displayIfNoReferences);
         }
 
+        // Kept around for binary compat with F#/TypeScript.
```

Comment:
> Does the TS build depend on explicit versions of Roslyn builds

TS will build against what we insert into devdiv.  As such, when we make an API change, it will result in an immediate source break for them once we insert (which happens 1-2 times a day).